pkg/kubelet/apis/config/types.go

```diff
@@ -347,6 +347,22 @@ type KubeletConfiguration struct {
 	// The purpose of this format is make sure you have the opportunity to notice if the next release hides additional metrics,
 	// rather than being surprised when they are permanently removed in the release after that.
 	ShowHiddenMetricsForVersion string
+
+	/* Experimental flags */
+
+	// lockFilePath is the path that kubelet will use to as a lock file.
+	// It uses this file as a lock to synchronize with other kubelet processes that may be running.
+	LockFilePath string
```

Comment:
The linked issue says for experimental flags "Below is the list of flags that needs to be evaluated and can be marked into categories like deprecated, removed. The remaining flags can be safely migrated to a stable config file format."

Have these experimental feature flags been evaluated?  NodeStatusMaxImages seems like a reasonable candidate to graduate, but i'm not sure about the others above...

Reply:
What is the better way to assess if the flag is a candidate to graduate? And what makes the flag Alpha yet, besides the *<Warning: Alpha feature>* in the description.

Reply:
It probably requires digging into the initial reason why they were added, and either asking the people who added the flags, or bringing it up at sig-node.

Reply:
alright, fair enough.